PowerUp/app/src/main/java/powerup/systers/com/IntroActivity.java

```diff
@@ -0,0 +1,70 @@
+package powerup.systers.com;
+
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.graphics.Color;
+import android.os.Bundle;
+
+import com.github.paolorotolo.appintro.AppIntro;
+import com.github.paolorotolo.appintro.AppIntroFragment;
+
+public class IntroActivity extends AppIntro {
+
+    SharedPreferences settings;
+    boolean firstRun;
+    @Override
+    public void init(Bundle savedInstanceState) {
+
+        settings = getSharedPreferences("prefs", 0);
+        firstRun = settings.getBoolean("firstRun", true);
+        if (firstRun) {
+
+            IntroActivity1 first_fragment = new IntroActivity1();
```

Comment:
Why do you create variables for this? Why don't you write `addSlide(new IntroActivity2())`instead?